vulcan/lib/client/jsx/contexts/session_sync.tsx

```diff
@@ -47,7 +49,7 @@ export function useSessionSync(
     return () => cancellable.cancel();
   }, [lastPollingRequest, dispatch, pollStatus, postInputs, state]);
 
-  const statusIsFresh = lastCompletedPollingRequest === state.current.session.inputs;
+  const statusIsFresh = _.isEqual(lastCompletedPollingRequest, state.current.session.inputs);
```

Comment:
aaaaangh, this change worries me alot  and could potentially really bite us from a performance point of view.  Is there a known case where we _need_ the deep comparison? Lately, I've been thinking to change how the session sync works to make this go away either way.

Reply:
Yes, because if you don't do the deep comparison, it will poll constantly.